### PR TITLE
Breaking: #97135 - web-info - Removed support for module handling based on TBE_M…

### PR DESCRIPTION
**Pull request by linawolf**
…ODULES_EXT

References https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/3 Releases: main, 12.4

### CODE REVIEW

**Comment by linawolf** [comment]
the provided code for migration does not seem to work currently: https://forge.typo3.org/issues/103627. Should we make a note?

File: Documentation/PageTsconfig/Mod/WebInfo.rst [hunk 1]
```diff
@@ -74,41 +74,24 @@ Example: Override the field definitions in the info module
...
-    ..  figure:: /Images/ManualScreenshots/Info/FunctionMenuInfoModule.png
-        :alt: The function menu of the Info module
+..  _pageblindingfunctionmenuoptions-webinfo-migration:
 
-        The function menu of the Info module
+Migration from menu.function to options.hideModules
+---------------------------------------------------
 
-    ..  warning::
+..  todo: link to options.hideModules once it is documented
```
Comment (brotkrueml) [inline Documentation/PageTsconfig/Mod/WebInfo.rst:86]: See here: https://docs.typo3.org/m/typo3/reference-tsconfig/12.4/en-us/UserTsconfig/Options.html#hidemodules

File: Documentation/PageTsconfig/Mod/WebInfo.rst [hunk 2]
```diff
@@ -74,41 +74,24 @@ Example: Override the field definitions in the info module
...
-        TYPO3\CMS\Belog\Module\BackendLogModuleBootstrap = 0
-        # Disable item "Pagetree Overview"
-        TYPO3\CMS\Info\Controller\PageInformationController = 0
-        # Disable item "Localization Overview"
-        TYPO3\CMS\Info\Controller\TranslationStatusController = 0
-        # Disable item "Linkvalidator"
-        TYPO3\CMS\Linkvalidator\Report\LinkValidatorReport = 0
-    }
+You can find the names of all TypoScript modules in
+:file:`EXT:info/Configuration/Backend/Modules.php`.
```
Comment (brotkrueml) [inline Documentation/PageTsconfig/Mod/WebInfo.rst:97]: Please use t3src for easy access. And I am not sure, that TypoScript modules are registered in EXT:info.

**Review by brotkrueml — CHANGES_REQUESTED** [review]

File: Documentation/PageTsconfig/Mod/WebInfo.rst (hunk 2 above)
Comment (linawolf) [reply Documentation/PageTsconfig/Mod/WebInfo.rst:97]: We are in the web_info module here and it is registered there, looked it up

File: Documentation/PageTsconfig/Mod/WebInfo.rst (hunk 1 above)
Comment (linawolf) [reply Documentation/PageTsconfig/Mod/WebInfo.rst:86]: Yes, it is quite counter intuitive as it seems like a page tsconfig option was superseeded by one in user tsconfig. We had some discutssions here: https://forge.typo3.org/issues/103627 and here: https://forge.typo3.org/issues/103627

File: Documentation/PageTsconfig/Mod/WebInfo.rst (hunk 2 above)
Comment (brotkrueml) [reply Documentation/PageTsconfig/Mod/WebInfo.rst:97]: So we have two options.hideModules, one in page TSconfig and one in user TSconfig?

File: Documentation/PageTsconfig/Mod/WebInfo.rst (hunk 1 above)
Comment (brotkrueml) [reply Documentation/PageTsconfig/Mod/WebInfo.rst:86]: ok

**Comment by linawolf** [comment]
@brotkrueml please have another look

File: Documentation/PageTsconfig/Mod/WebInfo.rst [hunk 3]
```diff
@@ -74,41 +74,30 @@ Example: Override the field definitions in the info module
...
 
-        Blinding the function menu items is not hardcore access control! All it
-        does is hide the possibility of accessing that module functionality
-        from the interface. It might be possible for users to hack their way
-        around it and access the functionality anyways. You should use the
-        option of blinding elements mostly to remove otherwise distracting options.
-
-..  _pageblindingfunctionmenuoptions-webinfo-example:
+..  code-block:: typoscript
+    :caption: **Page** Tsconfig, for example EXT:my_extension/Configuration/page.tsconfig
```
Comment (brotkrueml) [inline Documentation/PageTsconfig/Mod/WebInfo.rst:91]:
```suggestion
    :caption: **Page** TSconfig, for example EXT:my_extension/Configuration/page.tsconfig
```

File: Documentation/PageTsconfig/Mod/WebInfo.rst [hunk 4]
```diff
@@ -74,41 +74,30 @@ Example: Override the field definitions in the info module
...
 
-Example: Remove some options from the functions menu
-----------------------------------------------------
+    # before
+    mod.web_info.menu.function.TYPO3\CMS\Info\Controller\TranslationStatusController = 0
 
 ..  code-block:: typoscript
-    :caption: EXT:site_package/Configuration/page.tsconfig
+    :caption: **User** Tsconfig, for example EXT:my_extension/Configuration/user.tsconfig
```
Comment (brotkrueml) [inline Documentation/PageTsconfig/Mod/WebInfo.rst:97]:
```suggestion
    :caption: **User** TSconfig, for example EXT:my_extension/Configuration/user.tsconfig
```

File: Documentation/PageTsconfig/Mod/WebTs.rst [hunk 5]
```diff
@@ -1,48 +1,52 @@
...
+Migration from menu.function to options.hideModules
+---------------------------------------------------
 
-      The function menu of the Template module
+Migrate former usage of :tsconfig:`mod.web_ts.menu.function` in **page** TSconfig
+to option :ref:`options.hideModules <useroptions-hideModules>`.
 
-   .. warning::
+..  code-block:: typoscript
+    :caption: **Page** Tsconfig, for example EXT:my_extension/Configuration/page.tsconfig
```
Comment (brotkrueml) [inline Documentation/PageTsconfig/Mod/WebTs.rst:38]:
```suggestion
    :caption: **Page** TSconfig, for example EXT:my_extension/Configuration/page.tsconfig
```

File: Documentation/UsingSetting/UserTSconfig.rst [hunk 6]
```diff
@@ -200,27 +200,26 @@ When a page TSconfig property is set in **user** TSconfig that way, regardless
...
-user would still be able to select this menu item because the value of his user TSconfig
-overrides the same value set in the page TSconfig, just prefixed with
-:typoscript:`page.`:
+If we activate the following configuration in the user TSconfig of a certain backend
+user group, new and copied pages will be hidden for that group. The user TSconfig to
+be used is the same, but prefixed with :tsconfig:`page.`
 
 ..  code-block:: typoscript
-    :caption: EXT:site_package/Configuration/user.tsconfig
+    :caption: EXT:site_package/Configuration/TSconfig/editors.tsconfig
```
Comment (brotkrueml) [inline Documentation/UsingSetting/UserTSconfig.rst:217]: Why change the caption? user.tsconfig is the preferred way in v13+, as this is added automatically and cached (without manual import).

For v12 backport you can change this back if you like, but then use "TsConfig" in path as outlined here:
file:///home/chris/Projects/TYPO3CMS-Reference-CoreApi/Documentation-GENERATED-temp/ExtensionArchitecture/FileStructure/Configuration/TsConfig/Index.html

**Review by brotkrueml — CHANGES_REQUESTED** [review]

File: Documentation/UsingSetting/UserTSconfig.rst (hunk 6 above)
Comment (linawolf) [reply Documentation/UsingSetting/UserTSconfig.rst:217]: It is the preferred way to add **global** user TSconfig. However in this example only the TSconfig for a certain group should be set.

Comment (brotkrueml) [reply Documentation/UsingSetting/UserTSconfig.rst:217]: Yup, do it already this way in v12 -> condition. So, wrap it around a condition and everything is fine.

Comment (linawolf) [reply Documentation/UsingSetting/UserTSconfig.rst:217]: You can set the user TS config globally, but you can also still set it on group basis if that fits the use case. Setting user TS config directly in the group is neither deprecated nor discouraged. The only thing that got replaced was setting global / default user TSconfig.

Comment (brotkrueml) [reply Documentation/UsingSetting/UserTSconfig.rst:217]: Okay, I don't care about it anymore.

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**